### ISSUE TITLE
improved support for annotation of individual TKLBAM backups?

### ISSUE THREAD

**Issue by yosimor** [turnkeylinux/tracker#291]
Per my [post in the forum](http://www.turnkeylinux.org/forum/support/20150102/tklbam-backups-annotations-forksbranches), it would be highly desirable to be able to __annotate__ individual TKLBAM backups (with, say, the current revision-level of the project, or a short summary of what has changed since the last backup).

Currently, it is possible to add such annotations via Webmin, only _after_ the backup has been created [using:  System -> Backup and Migration (TKLBAM) -> View Log -> {individual backup} -> "Action Details" : "Log entry annotation"], but there is no easy way to actually _see_ the annotation.

I propose adding the ability to:

1. somehow specify this annotation __up-front__, using the Webmin interface and/or the [TKLBAM command-line](http://www.turnkeylinux.org/docs/tklbam/man).
2. somehow __see__ this annotation when listing the current set of backups (in [the Hub](https://hub.turnkeylinux.org/backups/), the Webmin interface, and using the [tklbam-list](http://www.turnkeylinux.org/docs/tklbam/man/tklbam-list) command-line)

**Comment by l-arnold** [comment]
I agree with Yosi. You can also make some annotations in TKL Hub, but mainly about the whole backup set in my recollection. 

I didn't know you could actually "Log Entry Annotation". Worth looking at for sure. 

best, 
Landis 
----- Original Message -----

**Comment by yosimor** [comment]
I suggest also adding tags for `webmin` and `hub` as well.

**Comment by JedMeister** [comment]
:+1:  Done :smile:

**Comment by lirazsiri** [comment]
Alright. I'll look into this for the next TKLBAM maintenance round.
